src/__test__/sample_v3/E3.las

```diff
@@ -0,0 +1,239 @@
+#---
+# File created using
+#---
+
+~Version
+ VERS.                  3.0   : LAS Version Identifier
+ WRAP.                   NO   : WRAP mode - one line per depth step
+ DLM .                  TAB   : Column Data Section Delimiter
+
+~Well
+ STRT.ft               2442.0   : First index value       {F}
+ STOP.ft                  0.0   : Last index value        {F}
+ STEP.ft                 -0.0   : STEP of index
+ NULL.              -999.25   : NULL value
+
+ API.       11-222-33333   : API Number
+ CNTY.              London   : County
+ COMP.    XYZ Company, LLC   : Oil Company
+ CTRY.                  UK   : Country
+ DATE.          11 Jun 2023   : Spud Date
+ EEL.ft                131.5   : Ground Level Elevation
+
+ LONG.deg           -92.3280   : Longitude
+ LONGX.           1507348.00   : X (Easting)
+ PLAN.                    3   : Directional Well Plan Number
+ RANG.                  12W   : Range
+ RIG.        RIG 486   : Rig Name
+ SEC.                  114   : Section
+ SRVC. Drilling Technologies,LLC   : Service Company
+ SVY_APPROVE.              Enabled   : Survey Pop-Up
+ TIME.           UK   : Local Time Zone
+ VS_AZM.deg                120.5   : Vertical Section Azimuth
+ WELL.  WELL ST112   : Well Name
+
+#---
+# The following 4 missing fields
+#---
+ LOC.                        : Location
+ GDAT.                        : Geodetic Datum
+ UWI.                        : Unique Well Identifier
+ PROV.                        : Province
+
+
+~Parameter
+#MNEM.UNIT              Value                              Description
+#---------  ----------------------------------------   -----------------------------------
+RUN .                                         1  : Run number
+GL  .Ft                             1363.000000  : Ground level elevation
+DREF.                                        KB  : Depth reference
+
+~Other

```

Comment:
I thought there is no ~Other section is v3 specification?

Reply:
I did not find an example with `~Other`, but I don't see a problem because ~Other is optional.

Reply:
Should I remove this ~Other sections from examples?

Reply:
Yeah that's alright cos ~Other is removed in v3